src/components/user/SignUpForm/index.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Field, reduxForm } from 'redux-form/immutable';
+import { View, Text, TouchableHighlight } from 'react-native';
+
+import Input from '../../common/Input';
+import * as constraints from '../../../utils/constraints';
+import styles from './styles';
+import CustomPicker from '../../common/Picker';
+
+const genderOptions = [
```

Comment:
What about moving these to a constants file? Maybe you want to use them somewhere else in the future.
And maybe have each object in a line to save file length? They're short enough.

`{ label: 'MALE', value: 'male' },`